index.html

```diff
@@ -1,305 +1,383 @@
-              </a>
-            </li>
-          </ul>
-          <span class="span-JS">JavaScript</span>
-        </div>
-
-        <div class="div-JS">
-          <ul class="icons-languages-div">
-            <li>
-              <a href="#">
-                <img class="JS languages-icons" src="Ellipse 2.svg" alt="" />
-              </a>
-            </li>
-          </ul>
-          <span class="span-JS">HTML</span>
+        <div class="Frameworks-div">
+          <h2 class="Frameworks">Frameworks</h2>
+          <div class="Arrow-frameworks">></div>
         </div>
-
-        <div class="div-JS">
-          <ul class="icons-languages-div">
-            <li>
-              <a href="#">
-                <img class="JS languages-icons" src="Ellipse 3.svg" alt="" />
-              </a>
-            </li>
-          </ul>
-          <span class="span-JS">CSS</span>
+        <div class="Skills-div">
+          <h2 class="Skills">Skills</h2>
+          <div class="Arrow-skills">></div>
         </div>
+      </section>
 
-      </div>
-      <br />
-      <div class="Frameworks-div">
-        <h2 class="Frameworks">Frameworks</h2>
-        <div class="Arrow-frameworks">></div>
-      </div>
-      <div class="Skills-div">
-        <h2 class="Skills">Skills</h2>
-        <div class="Arrow-skills">></div>
-      </div>
-    </section>
-  </main>
-</body>
+      
+      <!-- /////////footer///////////// -->
 
-</html>
\ No newline at end of file
+      <footer class="Contact_form_container">
+        <div class="div-imges">
+          <img class="img-1" src="top-1.png" alt="" />
+          <img class="img-2" src="top2.png" alt="" />
+          <img class="img-3" src="top-3.png" alt="" />
+        </div>
+        <div class="Contact_form">
+          <div class="form_heading">
+            <h2>Contact me</h2>
+          </div>
+          <p>
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+          <form class="form" action="https://formspree.io/f/{form_id}" method="post">
+            <ul class="footer-ul">
+              <li>
+                <input
+                  class="user_name"
+                  name="User_Name"
+                  type="text"
+                  required
+                  maxlength="30"
+                  placeholder="User Name"
+                />
+              </li>
+              <li>
+                <input
+                  class="user_email"
+                  name="User_Email"
+                  type="email"
+                  required
+                  placeholder="User Email"
+                  />
+                </li>
+                <li>
+                  <textarea
+                  class="massage"
+                  name="massage"
+                  placeholder="write your massage here"
+                  maxlength="300"
```

Comment:
- [ ] I can see that you implemented the requested changes by the previous reviewer which is awesome but it will be better if you increase the max length to "500" to meet the below requirement 👇 

- > Message field accepts up to 500 chars.